consensus/src/execution_ctx.rs

```diff
@@ -485,13 +517,7 @@ impl<'a, T: Operations + 'static, DB: Database> ExecutionCtx<'a, T, DB> {
                 );
                 if ret.is_ok() {
                     // Re-publish a drained message
-                    debug!(
-                        event = "republish",
-                        src = "future_msgs",
-                        msg_step = msg.get_step(),
-                        msg_round = msg.header.round,
-                        msg_topic = ?msg.topic(),
-                    );
+                    log_msg("outbound send", "future_msgs", &msg);
```

Comment:
I would like re-propagation to have a different event name than "new" outbound sends.
Can be address in follow-up PR